sopel/plugins/jobs.py

```diff
@@ -27,7 +27,7 @@
 
 
 class Scheduler(jobs.Scheduler):
-    """Plugins's Job Scheduler
+    """Plugins's Job Scheduler.
```

Comment:
```suggestion
    """Plugin job scheduler.
```

I don't think it needs to be a proper noun. 😅 At the very least, that weird possessive form needed to go!

Also, since I can't attach a note below (thanks for the line-note limitations, GitHub): What is meant by "their plugins" on L43? Maybe "its manager's plugins"? I must have missed that one in the giant review that was #1479, or whatever introduced this docstring.

Reply:
> This class is a specific implementation of the scheduler,
> made to store jobs by their plugins and be used by the bot (its ``manager``).

```
self._jobs[job.get_plugin_name()].append(job)
```

And "its manager" => "job scheduler's manager is the bot".